### PR TITLE
Flow support

### PR DESCRIPTION
**Pull request by hyochan**
* Add `types` path to `package.json`.
* Add flow support.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,11 +16,13 @@
     "notification"
   ],
   "main": "lib/index.js",
+  "types": "lib/index.d.ts",
   "files": [
     "lib"
   ],
   "scripts": {
-    "build": "tsc",
+    "flowgen": "flowgen lib/index.d.ts -o lib/index.js.flow && flowgen lib/src/toast-banner-component.d.ts -o lib/src/toast-banner-component.js.flow && flowgen lib/src/toast-banner-container.d.ts -o lib/src/toast-banner-container.js.flow",
```
Comment (benevbright) [inline package.json]: Could you check if there is a way to do it on all files, not manually? 
Considering this module is very beta and will have more files soon.

**Review by benevbright — CHANGES_REQUESTED** [review]
Thanks! 🙏🙏🙏 Happy Holiday :)

File: package.json (hunk 1 above)
Comment (hyochan) [reply package.json]: There isn't a possible solution in `flowgen`. Therefore I've made the script `flow.sh` :)

Comment (benevbright) [reply package.json]: Awesome! Thanks 🙌

**Review by benevbright — APPROVED** [review]
What a great Christmas Gift 👍

**Pull request merged**